pyproject.toml

```diff
@@ -20,16 +20,19 @@ requires-python = ">=3.10"
 
 # The core dependencies of the project.
 dependencies = [
```

Comment:
Note the changes here @ocefpaf. Probably should be synchronised into https://github.com/conda-forge/pyudunits2-feedstock/blob/main/recipe/meta.yaml